packages/core/src/components/ChoiceList/Choice.jsx

```diff
@@ -170,6 +174,10 @@ Choice.propTypes = {
    * otherwise, use the `checked` property.
    */
   defaultChecked: PropTypes.bool,
+  /**
+   * Access a reference to the `input` element
+   */
+  inputRef: PropTypes.func,
```

Comment:
I filed a ticket to standardize the naming for all these `ref` props across all the components

https://jira.cms.gov/projects/WNMGDS/issues/WNMGDS-242

Reply:
@pwolfert @line47 Do either of you have ideas for how to standardize these `ref` props? `fieldRef` is ok, but not great considering a button isn't a field. Naming after the actual html element doesn't work great either because this is often dynamic (i.e. <Textfield> could be `textarea` or `input`). In my head it would be nice to be able to use a single naming standard for accessing the base html element of a component, my top two are `inputRef` or `elemRef`

Reply:
Yeah, I'm not sure. If we really want it standardized, `inputRef` is reasonable even if it's a textarea, whereas `elemRef` is kind of an abiguous name because there are several elements in most of these components

Reply:
`inputRef` it is!